src/components/menu-item/menu.stories.js

```diff
@@ -49,12 +49,28 @@ export default {
 			description: 'Content inside the Menu Item',
 			control: { type: 'text' },
 		},
+		showArrowOnHover: {
```

Comment:
Could you write an example of the use of this prop to understand it better? Currently, it is not working in the storybook's existing example code.

Reply:
Yes it is there in storybook and readme.md

Reply:
We could mention in the description that when the menu items section is collapsed, the icon will be hidden and will show on hover if this option is enabled.